sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorLiveTestBase.cs

```diff
@@ -22,7 +22,7 @@ public MetricsAdvisorLiveTestBase(bool isAsync) : base(isAsync)
             Sanitizer = new MetricsAdvisorRecordedTestSanitizer();
         }
 
-        internal const string DetectionConfigurationId = "fb5a6ed6-2b9e-4b72-8b0c-0046ead1c15c";
+        internal const string DetectionConfigurationId = "35c5ee94-8512-4cdc-afc0-fdbb36e38569";
```

Comment:
Reason for updating it explained here: https://github.com/Azure/azure-sdk-for-net/pull/21505#discussion_r643888295. Forgot to update this one.

Reply:
By the way, this implied in a LOT of rerecordings since this is not an env var (and not stored as part of the recording itself).

Reply:
I'll revert this change for the reasons specified in https://github.com/Azure/azure-sdk-for-net/pull/21505#discussion_r643888295.